### PR TITLE
Feature/ Create form for adding internship program

### PR DESCRIPTION
**Pull request by svtslv17**
Screenshot_3ggggggggggggggg
Getting location data from the backend. Language and technology stack data is currently in mock data. The formation of the data object of the internship is carried out by the Formic library. At the moment, there is no sending to the backend until the backend bugs are fixed. The data is output to the browser console.

### CODE REVIEW

File: src/components/addProgram/AddProgram.jsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      const newInternship = { ...values };
+      newInternship.locations = newInternship.locations.map((country) => {
+        const countryObject = { name: country };
+        return countryObject;
+      });
+      newInternship.internshipStacks = newInternship.internshipStacks.map((stack) => {
+        const stackObject = { technologyStackType: stack };
+        return stackObject;
+      });
+      console.log(JSON.stringify(newInternship, null, 2));
```
Comment (itsnotpolza) [inline src/components/addProgram/AddProgram.jsx:55]: remove console statements

File: src/components/addProgram/AddProgram.jsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    },
+  });
+
+  const dataForRenderDatePicker = [
+    // [keyName, label]
```
Comment (itsnotpolza) [inline src/components/addProgram/AddProgram.jsx:60]: why is this an array and not object

File: src/components/cardTitle/CardTitle.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box, Typography } from '@mui/material';
+// import { useMediaBetween } from '../utils';
```
Comment (itsnotpolza) [inline src/components/cardTitle/CardTitle.jsx:3]: remove comment

File: src/components/cardTitle/CardTitle.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import './CardTitle.sass';
+
+const CardTitle = ({ title }) => (
```
Comment (itsnotpolza) [inline src/components/cardTitle/CardTitle.jsx:6]: is this component necessary?

File: src/components/sidePopUp/SidePopUp.sass [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: none
+  z-index: 2
+  top: 0
+  right: 0
+  display: flex
+  align-items: center
+  justify-content: center
+  z-index: 1101
+  transform: scale(0)
+  transition: background-color 0.6s ease-in-out;
```
Comment (itsnotpolza) [inline src/components/sidePopUp/SidePopUp.sass:14]: remove semicolons

**Review by itsnotpolza — APPROVED** [review]

**Review by ValeryiaMIRON — APPROVED** [review]

**Review by nsukhotskaya — APPROVED** [review]

**Pull request merged**